docs/hello_nextflow/01_hello_world.md

```diff
@@ -0,0 +1,686 @@
+
+The actual output of the `sayHello` process is `output.txt`.
+Open it and you will find the `Hello World!` greeting, which was the expected result of our minimalist workflow.
+
+```console title="output.txt" linenums="1"
+Hello World!
+```
+
+### Takeaway
+
+You know how to decipher a simple Nextflow script, run it and find the output and relevant log files in the work directory.
+
+### What's next?
+
+Learn how to manage your workflow executions conveniently.
+
+---
+
+## 3. Manage workflow executions
+
+Knowing how to launch workflows and retrieve outputs is great, but you'll quickly find there are a few other aspects of workflow management that will make your life easier, especially if you're developing your own workflows.
+
+Here we show you how to use the `publishDir` directive to TODO, the `resume` feature for when you need to re-launch the same workflow, and how to delete older work directories with `nextflow clean`.
+
+### 3.1. Publish outputs
+
+As you have just learned, the output produced by our pipeline is buried in a working directory several layers deep.
+This is done on purpose; Nextflow is in control of this directory and we are not supposed to interact with it.
+
+However, that makes it inconvenient to retrieve outputs that we care about.
+
+Fortunately, Nextflow provides a way to manage this more conveniently, called the `publishDir` directive, which acts at the process level.
+This directive tells Nextflow to copy the output(s) of the process to a designated output directory.
+It allows us to retrieve the desired output file without having to dig down into the work directory.
+
+#### 3.1.1. Add a `publishDir` directive to the `sayHello` process
+
+In the workflow script file `hello-world.nf`, make the following code modification:
+
+_Before:_
+
+```groovy title="hello-world.nf" linenums="6"
+process sayHello {
+
+    output:
+        path 'output.txt'
+```
+
+_After:_
+
+```groovy title="hello-world.nf" linenums="6"
+process sayHello {
+
+    publishDir 'results', mode: 'copy'
+
+    output:
+        path 'output.txt'
+```
+
+#### 3.1.2. Run the workflow again
+
+Now run the modified workflow script:
+
+```bash
+nextflow run hello-world.nf
+```
+
+The log output should look very familiar:
+
+```console title="Output" linenums="1"
+ N E X T F L O W   ~  version 24.10.0
+
+Launching `hello-world.nf` [jovial_mayer] DSL2 - revision: 35bd3425e5
+
+executor >  local (1)
+[62/49a1f8] sayHello | 1 of 1 ✔
+```
+
+This time, Nextflow has created a new directory called `results/`.
+Our `output.txt` file is in this directory.
+If you check the contents it should match the output in the work subdirectory.
+This is how we move results files outside of the working directories conveniently.
+
+It is also possible to set the `publishDir` directive to make a symbolic link to the file instead of actually copying it.
+This is preferable when you're dealing with very large files you don't need to retain longer term.
```

Comment:
```suggestion
When you're dealing with very large files that you don't need to retain for long, you may prefer to set the `publishDir` directive to make a symbolic link to the file instead of copying it.
```